packages/ui/src/assets/css/googleButton.css

```diff
@@ -1,5 +1,9 @@
+:root {
+  --background-color: var(--dz-primary-color)
+}
+
 .google.dark {
-  background-color: #4285f4 !important;
+  background-color: var(--background-color) !important;
```

Comment:
You can keep it as it was before. This is specific to google provider so we cannot use the background-color here. Also background-color is too generic name here. Lets revert back to original implementation here.